config/.env.example

```diff
@@ -1,18 +1,20 @@
-NODE_ENV=local
-PORT=3000
-API_ROOT_URL=http://localhost:3000
-FRONT_BASE_URL=http://localhost:4200
+# Secrets
+# YOU MUST CHANGE THESE BEFORE GOING INTO PRODUCTION
+JWT_SECRET=your-secret
 
-MONGO_URL=mongodb://localhost:27017/notifire-db
-REDIS_PORT=6379
-REDIS_HOST=localhost
+## General
+NODE_ENV=test
```

Comment:
Should we use the test env here or maybe create a new one called docker? or just use prod/production.

Reply:
I think that the user should have the ability to choose what environment he wants whether it is production or test as it was without any containers at all, and that way we will not have the extra environment as docker or even docker_test /docker_production.
@scopsy do you think it is a good idea to create one called docker for the user?

Reply:
I think we better stick to local/dev and prod, because we might do specific optimizations based on that. So maybe let's just change the default to local here and let the user decide to choose between local and prod.

The test env is needed only for development purposes to run our test suite with. The docker-compose users don't really need to access it at all.

Reply:
We are on the same page on that one.
I probably did not understand correctly what you wrote above I will update the default to local.

Reply:
Now reading my first comment, I find it hard to understand myself as well 😅